src/ssl.c

```diff
@@ -239,6 +239,45 @@ byte tsip_rootCAverified( );
 #endif
 
 #ifdef WOLFSSL_SESSION_EXPORT
+/**
+ * Used to import a serialized TLS session. In most cases wolfSSL_get_session
+ * should be used instead, this function is for exporting the state of the
+ * connection and when imported a resumption is not needed.
+ * WARNING: buf contains sensitive information about the state and is best to be
+ *          encrypted before storing if stored.
+ *
+ * @param ssl WOLFSSL structure to import the session into
+ * @param buf serialized session
+ * @param sz  size of buffer 'buf'
+ * @return the number of bytes read from buffer 'buf'
+ */
+int wolfSSL_tls_import(WOLFSSL* ssl, const unsigned char* buf, unsigned int sz)
+{
+    if (ssl == NULL || buf == NULL) {
+        return BAD_FUNC_ARG;
+    }
+    return wolfSSL_session_import_internal(ssl, buf, sz, WOLFSSL_EXPORT_TLS);
+}
+
+
+/**
```

Comment:
Same... The doxy style comments for API can be removed since they are in `doc/dox_comments/header_files/ssl.h`. Otherwise we are duplicating language that needs to be updated.